tenseal/__init__.py

```diff
@@ -11,6 +11,7 @@
 SecretKey = _ts_cpp.SecretKey
 RelinKeys = _ts_cpp.RelinKeys
 GaloisKeys = _ts_cpp.GaloisKeys
+BigUInt = _ts_cpp.BigUInt
```

Comment:
Everything that relates to seal_api should be under `tenseal.seal_api` namespace and not directly in `tenseal`